js/layout-javascript/agenda-code.js

```diff
@@ -706,10 +706,16 @@ DynamicList.prototype.initialize = function() {
       return;
     })
     .then(function() {
-      return Fliplet.DataSources.getById(_this.data.dataSourceId);
+      if (_this.data.detailViewAutoUpdate && Fliplet.Navigator.isOnline()) {
+        return Fliplet.DataSources.getById(_this.data.dataSourceId);
```

Comment:
I would suggest a `.catch(function () { return Promise.resolve(); });`

Reply:
But if the error comes from somewhere else it will resolve too.

Reply:
I thin he meant adding that to the `getById` call. Right?

Reply:
@hcarneiro I'm suggesting catching this promise, not others.

```js
return Fliplet.DataSources.getById(_this.data.dataSourceId).catch(function () {
  return Promise.resolve(); // no problem when this fails
});
```